test/cli/test_cassettes.py

```diff
@@ -153,14 +154,16 @@ async def test_replay(openapi_version, cli, schema_url, app, reset_app, cassette
 
 
 def test_multiple_cookies(base_url):
-    response = requests.get(f"{base_url}/success", cookies={"foo": "bar", "baz": "spam"})
+    headers = {"User-Agent": USER_AGENT}
+    response = requests.get(f"{base_url}/success", cookies={"foo": "bar", "baz": "spam"}, headers=headers)
     request = Request.from_prepared_request(response.request)
     serialized = {
         "uri": request.uri,
         "method": request.method,
         "headers": request.headers,
         "body": {"encoding": "utf-8", "base64_string": request.body},
     }
+    assert USER_AGENT in serialized["headers"]["User-Agent"]
```

Comment:
```suggestion
    assert serialized["headers"]["User-Agent"] == USER_AGENT
```

I think that comparing the whole string is better than comparing if a substring is present in this case

Reply:
It true, but serialized["headers"]["User-Agent"] is a tuple.

Reply:
You are right, my bad

Reply:
No problem :)